homework41/src/EmojiList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Emoji from "./Emoji";
+
+const EmojiList = ({ emojis, incrementCount }) => {
+    
+    if (!Array.isArray(emojis)) {
+
+        return; 
```

Comment:
If you need this `if` block, then `return null;` or `return false;`

`return` => `return undefined` => react may throw errors